test/AutoSubscribe.spec.tsx

```diff
@@ -387,6 +397,44 @@ describe('AutoSubscribe', function () {
         expect(Component.state('keyedDataSum')).toEqual(13);
     });
 
+    it('autoSubscribeWithKey and key - test single-@key compound key Subscriptions', () => {
+        let expectedState = 1;
+        const Component = makeComponent({ test_compoundSingleKeySub: true, ids: ['a'] });
+        expect(Component.state('stateChanges')).toEqual(expectedState);
+        expect(Component.state('keyedDataSum')).toEqual(4);
+
+        SimpleStoreInstance.setStoreDataForCompoundEnumKeyedSubscription(['a'], TriggerKeys.First, 1);
+        expect(Component.state('stateChanges')).toEqual(++expectedState);
+        expect(Component.state('keyedDataSum')).toEqual(6);
+
+        SimpleStoreInstance.setStoreDataForCompoundEnumKeyedSubscription(['a'], TriggerKeys.Second, 1);
+        expect(Component.state('stateChanges')).toEqual(++expectedState);
+        expect(Component.state('keyedDataSum')).toEqual(7);
+
```

Comment:
Qhat about if only 'a' or TriggerKeys.First are triggered here? (and in next test)

Reply:
Should do nothing.

Reply:
Tests should validate this to 1) ratify behaviour 2) prevent regression

Reply:
Ya for sure.  Need several negative tests after this.